src/storage/src/hummock/compactor/mod.rs

```diff
@@ -371,6 +408,15 @@ impl Compactor {
                         tracing::info!("Compactor is shutting down");
                         return;
                     }
+                    // Handle failed tasks.
+                    task_results = &mut task_collector => {
```

Comment:
It seems that we only need to poll it in `consume_stream`. Because compactor would not create connection frequently

Reply:
> It seems that we only need to poll it in `consume_stream`. Because compactor would not create connection frequently

fixed